docs/Wallet.md

```diff
@@ -82,9 +82,19 @@ Warning: Despite the fact, in this example, that you entered `1.0` in the amount
 
 ##### RBF (Replace-by-fee)
 
+Replace-by-fee (RBF) is an advanced privacy-enhancing feature that allows you to replace a previously broadcast transaction (while unconfirmed) to randomize your wallet's transaction fingerprint, or simply for replacing it with a higher fee rate to move the transaction higher in the queue of confirmation (mining) priority. This will effectively replace the original transaction as the higher fee rate will be prioritized and once confirmed, invalidating the original one (double spend). 
```

Comment:
I would replace `is an advanced privacy-enhancing feature` with `is a Bitcoin protocol feature`